overrides/default.nix

```diff
@@ -1853,6 +1853,13 @@ lib.composeManyExtensions [
         }
       );
 
+      pycurl = super.pycurl.overridePythonAttrs (
+        old: {
+          propagatedBuildInputs = (old.propagatedBuildInputs or [ ]) ++ [ pkgs.curl ];
```

Comment:
This should not need to be propagated—after PycURL is built, the C headers for cURL are not needed to use PycURL.